dazzle.yaml

```diff
@@ -5,6 +5,15 @@ combiner:
         - dep-cacert-update
         - tool-docker
         - tool-tailscale
+    - name: gitpod-dev
+      ref:
+      - base
+      chunks:
+        - lang-c
+        - lang-go:1.20
+        - lang-java:11
+        - lang-node:18
+        - tool-brew
```

Comment:
Consider removing the first `gitpod-dev` combination, it's a duplicate of [this](https://github.com/gitpod-io/workspace-images/pull/1113/files#diff-b8b5d73ca5ee13982e492c43c0198eda34749eab5c1f8b3621f6512ec5af6f8bR190-R199)

Reply:
done